src/main/java/core/basesyntax/MyMap.java

```diff
@@ -6,4 +6,6 @@
     V getValue(K key);
 
     int getSize();
+
+    boolean isEmpty();
```

Comment:
have not need change interface by default
replace this method to MyHashMap class